go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/btcsuite/btclog
+
+go 1.22.2
```

Comment:
Since `slog` was added in `go 1.21`, perhaps we should use that to make this as far backward compatible as possible? AFAIK there are some projects out there that compile parts of the btcsuite for microprocessors (using Tinygo or similar tools) and can't quickly update to the latest version.